azconfig/client/pom.xml

```diff
@@ -12,55 +12,23 @@
         <version>1.0.0-SNAPSHOT</version>
         <relativePath>../../pom.client.xml</relativePath>
     </parent>
+
+    <groupId>com.microsoft.azure</groupId>
```

Comment:
groupId should be `com.azure` per [Java Spec](https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html)